ntbs-service/Migrations/20200316123418_ChangeSmokingStatusToRiskFactor.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class ChangeSmokingStatusToRiskFactor : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "SmokingStatus",
+                table: "SocialRiskFactors");
```

Comment:
We're loosing the data in here - should we build the new table, port over what we can, then drop this one?